### PR TITLE
Add Persian translation

### PR DESCRIPTION
**Pull request by faridprogrammer**
Hi.
I have added the Persian translation to the localization of the project. 
I think there is no need to make any changes in Front-End projects or MVC.
Am I right?

### CODE REVIEW

**Review by demirmusa — APPROVED** [review]

**Comment by demirmusa** [comment]
Yes @faridprogrammer, you are right. 
Thank you for your contribution.

**Comment by demirmusa** [comment]
Yes @faridprogrammer, I made some minor changes
Thank you for your contribution.

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Core/AbpProjectNameCoreModule.cs [hunk 1]
```diff
@@ -33,6 +34,8 @@ public override void PreInitialize()
             AppRoleConfig.Configure(Configuration.Modules.Zero().RoleManagement);
 
             Configuration.Settings.Providers.Add<AppSettingProvider>();
+            
+            Configuration.Localization.Languages.Add(new LanguageInfo("fa", "فارسی", "famfamfam-flags ir"));
```
Comment (faridprogrammer) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Core/AbpProjectNameCoreModule.cs:38]: Hi.
Thanks for the corrections you have made. 
Could you please explain why did you add this line? 
Why adding the language to DefaultLanguagesCreator was not enough? 

Thanks

Comment (demirmusa) [reply aspnet-core/src/AbpCompanyName.AbpProjectName.Core/AbpProjectNameCoreModule.cs:38]: Hi @faridprogrammer,
You can follow the steps to add a new language to your project from here https://aspnetboilerplate.com/Pages/Documents/Localization.

The reason you won't see the same implementation for other languages is probably because those implementations have already been made elsewhere in abp.

Comment (faridprogrammer) [reply aspnet-core/src/AbpCompanyName.AbpProjectName.Core/AbpProjectNameCoreModule.cs:38]: Thanks for the explanation.
